wdl-lint/src/rules/shellcheck.rs

```diff
@@ -375,6 +378,84 @@ impl<'a> CommandContext<'a> {
     }
 }
 
+/// Detect embedded quotes surrounding an expression in a string.
+/// For a given expression, it checks through all descendants to see if there
+/// are any name references (variables) that are surrounded by escaped quotes.
+/// In WDL, the parent expressions is an addition (concatenation) operations.
```

Comment:
what does this mean?

Reply:
However, the comment should probably be updated, because you can embed variables in strings like this one: ` ~{if read_group != "" then "-r '~{read_group}'" else ""}` by adding an extra expression block (`~{}`).